src/quo2/components/switcher_tabs.cljs

```diff
@@ -0,0 +1,83 @@
+(ns quo2.components.switcher-tabs
+  (:require [reagent.core :as reagent]
+            [quo.react :as react]
+            [quo.react-native :as rn]
+            [quo2.reanimated :as reanimated]
+            [quo2.components.text :as text]
+            [status-im.switcher.constants :as constants]
+            [oops.core :refer [oget]]))
+
+(def switcher-tabs-js (js/require "../src/js/switcher_tabs.js"))
+
+(def switcher-header-tab-padding-horizontal 10)
+
+(def tabs-data [{:name "Switch" :color "#40444C"}
+                {:name "Scan" :color "#0C1724"}
+                {:name "Share" :color "#40444C"}
+                {:name "Activity" :color "#40444C"}])
+
+(defn switcher-header-tab-text [translate-x tab-name tab-index]
+  [:f>
+   (fn []
+     (let [animated-style (.useSwitcherHeaderTextStyle ^js switcher-tabs-js translate-x tab-index)]
+       [text/text {:size   30
+                   :weight :semi-bold
+                   :animated-style animated-style} (str tab-name)]))])
```

Comment:
why `str` is needed ?

Reply:
Because otherwise `react-native-reanimated` can't find a matching `View Config Getter Callback`

Reply:
mhm interesting

Reply:
This way it's clear to REA, that the child element is a string, and not a view where there is no component for